gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-distribution/src/main/assembly/plugin-assembly-distribution.xml

```diff
@@ -132,8 +131,8 @@
                 <exclude>com.graviteesource.*:*:*</exclude>
                 <exclude>org.projectlombok:*:*</exclude>
                 <exclude>org.mapstruct:mapstruct-processor:*</exclude>
+                <exclude>*:*:*:windows-x86_64</exclude>
             </excludes>
-            <scope>compile</scope>
```

Comment:
Have you checked the consequence on the content of the lib/ext folder? And the size of the final bundle?

Reply:
I update the assembly accordingly, so shouldn't have any consequence.